server/src/main/java/fr/uga/l3miage/tp2/exo3/models/CommandeEntity.java

```diff
@@ -0,0 +1,24 @@
+package fr.uga.l3miage.tp2.exo3.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import java.util.Date;
+import java.util.Set;
+
+@Entity
+public class CommandeEntity {
+    @Id
+    private Long id;
+
+    private Date date;
+
+    private Double montantTotal;
+
+    @ManyToOne
+    private ClientEntity client;
+
+    @OneToMany(mappedBy = "commande")
+    private Set<ProduitEntity> produits;
```

Comment:
Alors oui effectivement vous avez fait un unidrectionnelles, cependant votre `mappedBy` ne sert à rien, ici il fallait utiliser `@JoinColumn`, car ici ça vous créer une table d'association, donc 
```suggestion
    @OneToMany(mappedBy = "commande")
    @JoinColumn(name = "id_produit",referencedColumnName = "id")
    private Set<ProduitEntity> produits;
```